src/lightning_app/components/serve/auto_scaler.py

```diff
@@ -574,5 +641,8 @@ def autoscale(self) -> None:
         self._last_autoscale = time.time()
 
     def configure_layout(self):
-        tabs = [{"name": "Swagger", "content": self.load_balancer.url}]
+        tabs = [
+            {"name": "Endpoint Info", "content": f"{self.load_balancer}/endpoint-info"},
+            {"name": "Swagger", "content": self.load_balancer.url},
```

Comment:
Not sure the swagger UI is needed, API access UI should cover it

Reply:
Load balancer also creates a few other endpoints for app developers to see how many concurrent requests are currently active etc. Swagger gives info about those as well